### DISCUSSION TITLE
Ebusd does not take local config files

### DISCUSSION THREAD

**Discussion by Flyrian** [john30/ebusd#1055] [Q&A]
Hi everyone,

first, thanks to John for the effort of creating this masterpiece of SW.

I read through all discussions i could find and tried everything I could find on this topic but nothing works.
My ebusd does not take the local configuration files. Can anyone please review my settings and tell me where my mistake is?
As you can see i would need to switch to the 15.ctlv2.csv, but it do not take any config file locally and i spent a frew hours now, but didnt get it...
Thanks upfront:

Config:
EBUSD_OPTS="--scanconfig=full --configpath=/home/pi/ebusd-configuration/ebusd-2.1.x/en --accesslevel=* -d ens:/dev/ttyAMA0 --latency=50 --httpport=8080 --address=ff"

:/ $ ebusctl i
version: ebusd 23.2.23.2
update check: OK, device firmware 1[3a0f] available
device: /dev/ttyAMA0, enhanced high speed, firmware 1.1[380f].1[380f]
access: *
signal: acquired
symbol rate: 23
max symbol rate: 213
min arbitration micros: 2
max arbitration micros: 356
min symbol latency: 4
max symbol latency: 14
scan: finished
reconnects: 0
masters: 4
messages: 146
conditional: 0
poll: 1
update: 10
address 03: master #11
address 04: slave #25, ebusd, scanning
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0901;HW=5103", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV2;SW=0514;HW=1104"
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0607;HW=5103"
address ec: slave, scanned "MF=Vaillant;ID=SOL00;SW=0514;HW=1104", loaded "vaillant/ec.sol.sc.csv"
address ff: master #25, ebusd

BR Florian

**Comment by plasma99new** [comment]
Hi,

I think you're looking in the wrong place. If this is the information about your instance started with the given config, then it has taken 2 local files. For one device, it just can't find a config or no device at all.

I'm experiencing a similar problem right now, and have had it in the past as well.

If I remember the solution, I'll be sure to get in touch.
